config/mail.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'driver' => env('MAIL_DRIVER', 'smtp'),
+    'driver' => env('MAIL_DRIVER', 'mandrill'),
```

Comment:
yep. I read something, cause I had changed it when I did the redis caching stuff too, in the app config, and saw it's a bad idea to change _there_ from the Laravel defaults. Cause if redis is not turned on it can at least default to a less ideal but still working driver.